placebo/serializer.py

```diff
@@ -53,8 +66,13 @@ def serialize(obj):
         result['second'] = obj.second
         result['microsecond'] = obj.microsecond
         return result
-    if isinstance(obj, StreamingBody):
-        result['body'] = obj.read()
+    # Convert objects to dictionary representation based on type
+    if isinstance(obj, botocore.response.StreamingBody):
+        result['payload'] = obj.read() 
+        # Set the original stream to the buffered representation of itself,
+        # so that it can be re-read downstream.
+        obj._raw_stream = StringIO(result['payload'])
```

Comment:
I'm surprised this works; I would have thought you would have needed a BytesIO.

Can you add additional tests for these cases to show they work in all versions of python that Travis tests?